samples/Playground/Playground.iOS/Playground.iOS.csproj

```diff
@@ -68,6 +68,9 @@
     <Reference Include="Xamarin.iOS" />
     <Reference Include="System.Numerics" />
     <Reference Include="System.Numerics.Vectors" />
+    <Reference Include="Plugin.Permissions">
+      <HintPath>..\packages\Plugin.Permissions.5.0.0-beta\lib\xamarinios10\Plugin.Permissions.dll</HintPath>
```

Comment:
Please don't use old approach for add nuget, use `<PackageReference />` in csproj